public/assets/js/monster/test-top-scenarios.js

```diff
@@ -121,3 +121,31 @@ crud.field('wish').subfield('country').change(function(field) {
     crud.field('repeatable_example_1').subfield('if_yes', field.rowNumber).show(field.value == 'yes').enable(field.value == 'yes'); 
 });
 
+ //EXAMPLE 12
+ // USING LIVE VALIDATION
```

Comment:
Please add one more comment line here, to explain what you mean by "live validation". 
Eg. When X then Y otherwise Z.

Reply:
Done. Let me know if it's enough!